nixos/modules/services/taler/module.nix

```diff
@@ -0,0 +1,66 @@
+{
+  lib,
+  pkgs,
+  config,
+  ...
+}:
+
+let
+  settingsFormat = pkgs.formats.ini { };
+  this = config.services.taler;
+in
+
+{
+  imports = [ ./exchange.nix ];
```

Comment:
Please move this up into modules-list.nix

Reply:
This has already been done (see [here](https://github.com/NixOS/nixpkgs/pull/332699/files#diff-7c411742b9908f3df53ed26638c28825bf5dc0886cd42f71e44c878783803829R1354))